### DISCUSSION TITLE
Data Export - Followers

### DISCUSSION THREAD

**Discussion by alisonw** [mastodon/mastodon#19940] [Q&A]
instance/settings/export has options to get .csv of your _follows_, _blocks_, _bookmarks_, etc. but not of your _followers_. Was this a design decision or an omission?

**Comment by johnfisherman** [comment]
I saw this PR that mentioned the fact that an export of followers could be way too heavy, in the case in which you are a rock star or a famous CEO. Think 300.000 followers. I'm sure there is a technical solution to this edge case, but I'm not going any deeper here.
